conferences/2018/javascript.json

```diff
@@ -651,6 +671,8 @@
     "endDate": "2018-07-20",
     "city": "Edinburgh",
     "country": "U.K.",
+    "cfpUrl": "https://www.papercall.io/scotlandjs-2018",
+    "country": "2018-02-28",
```

Comment:
looks like this is wrong

Reply:
It is but it's been fixed here: https://github.com/tech-conferences/javascript-conferences/pull/181/files

Reply:
oh cool :+1

Reply:
Thanks for the catch though! 😬